src/Jenssegers/Mongodb/Connection.php

```diff
@@ -267,6 +272,40 @@ protected function getDefaultSchemaGrammar()
         return new Schema\Grammar();
     }
 
+    public function beginTransaction(array $options = [])
```

Comment:
What is the expected behaviour if a transaction is already in progress?

Reply:
Hi @alcaeus , there is another PR which looks promising. If you'd like and have a time, please take a look https://github.com/jenssegers/laravel-mongodb/pull/1904

Thanks!